Dispatcher/Supporting Files/Config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "api_key": "oE0GNhaEFfOy9eAygqEaq4P9ir8hqxrIui7u20H5_e4"
```

Comment:
Need to add the Config.json to the .gitignore file so that it wouldn't be uploaded to github